src/__test__/redux/actions/cellSets/createCellSet.test.js

```diff
@@ -45,14 +51,15 @@ describe('createCellSet action', () => {
     expect(firstAction).toMatchSnapshot();
   });
 
-  it('Last action dispatches cellSetSave event', async () => {
+  it('Send fetch to the api', async () => {
```

Comment:
A bit more complete description : `Send fetch to the API when creating cell set`